clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -107,33 +107,37 @@ function extractMessagesFromGVLTranslations(
   // itself is not available in most languages
   const extracted: Record<Locale, Messages> = {};
   locales.forEach((locale) => {
+    // TODO (PROD-1811): prefer a case-insensitive locale match
```

Comment:
While coding this up I realized the right thing to do here is do some reasonably-fuzzy matching of GVL locales to our experience locales 😓

Reply:
Moved this to a DEFER since this is a blocker for testing!